src/algebra/indicator_function.lean

```diff
@@ -183,6 +183,34 @@ end
   (mul_indicator s f)⁻¹' B = s.ite (f ⁻¹' B) (1 ⁻¹' B) :=
 by letI := classical.dec_pred (∈ s); exact piecewise_preimage s f 1 B
 
+@[to_additive] lemma mul_indicator_one_preimage (s : set M) :
+  t.mul_indicator 1 ⁻¹' s ∈ ({set.univ, ∅} : set (set α)) :=
```

Comment:
Do you need this lemma? IMHO, a better approach is to rewrite `mul_indicator_one'` or `mul_indicator_one`.

Reply:
I do not need this lemma, but it seemed in the same spirit as the one that actually is used in LTE.